test/metabase/api/alert_test.clj

```diff
@@ -630,6 +659,23 @@
             (mt/user-http-request :rasta :put 403 (alert-url alert)
                                   (dissoc (default-alert-req card pc {} []) :channels))))))))
 
+(deftest update-alert-enable-http-channel-test
+  (mt/with-temp [:model/Pulse     alert (basic-alert)
+                 :model/Card      card  {}
+                 :model/PulseCard _     (pulse-card alert card)
+                 :model/Channel   {channel-id :id} {:type    :channel/http
+                                                    :details {:auth-method "none"
+                                                              :url         "https://metabasetest.com"}}]
+    (is (=? {:channels [{:channel_type "http"
```

Comment:
Can we wrap both of these assertions in "testing" blocks for clarity?

Reply:
yes 4b7f101c22